example-config.yaml

```diff
@@ -21,7 +21,7 @@ appservice:
         type: sqlite3
         # The database URI.
         #   SQLite: File name is enough. https://github.com/mattn/go-sqlite3#connection-string
-        #   Postgres: Connection string. For example, postgres://user:password@host/database
+        #   Postgres: Connection string. For example, uri: 'host=host user=user password=password dbname=database sslmode=disable'
```

Comment:
maybe include both strings, as either of them is working

Reply:
That depends on how you see it.

The reason I submitted the proposed change, is because you need to supply `sslmode=disable` when connecting to postgres:latest as a sidecar

So the earlier example, actually doesn't always/universally work - the proposed one presumably does.

Reply:
`?sslmode=disable` works with the old scheme

Reply:
Ahh, was not aware of that.

Well, the only thing I wanted to include was how to disable SSL - either scheme works for me.